apps-rendering/src/components/editions/galleryImage/index.tsx

```diff
@@ -59,9 +70,9 @@ const getCaptionDetails = (oDoc: Option<DocumentFragment>): CaptionDetails => {
 					? some(node.textContent)
 					: none;
 			} else if (node.nodeName === '#text') {
-				details.description = node.textContent
-					? some(node.textContent)
-					: none;
+				details = pushToDescription(details, node);
+			} else if (node.nodeName === 'A') {
+				details = pushToDescription(details, node);
```

Comment:
I'm wondering if both `if (node.nodeName === '#text')` and `if (node.nodeName === 'A')` are going to have the same implementation as `details = pushToDescription(details, node);` what's the reason to have them seperately?

Reply:
That's a very good point. Fixed now.